src/sickle.ts

```diff
@@ -263,6 +267,17 @@ class Annotator {
     this.emit(': void;\n');
   }
 
+  private visitEnum(node: ts.EnumDeclaration) {
+    this.emit('\n');
+    for (let member of node.members) {
+      this.emit('/** @export */ ');
```

Comment:
This shouldn't impact the size of the code at all. All this does is allow you to do `Foo.BAR` instead of having to do `Foo['BAR']`. The typescript enum won't get minified either way, because it uses `Foo['BAR']`.